pallets/phala/src/lib.rs

```diff
@@ -25,12 +25,15 @@ pub mod registry;
 use compute::{basepool, mining, pawnshop, poolproxy, stakepoolv2, vault};
 
 use frame_support::traits::LockableCurrency;
+/// The unified config of the compute pallets
 pub trait PhalaConfig: frame_system::Config {
 	type Currency: LockableCurrency<Self::AccountId, Moment = Self::BlockNumber>;
 }
+/// The unified type Balance of pallets in compute with template T.
```

Comment:
```suggestion
/// The unified type Balance of pallets from the runtime T.
```

Accurately speaking, `<T>` is the generic type parameter, not called a template (C++ specific).